lib/franklin/posts/projectors/post.ex

```diff
@@ -2,9 +2,7 @@ defmodule Franklin.Posts.Projectors.Post do
   use Commanded.Projections.Ecto,
     # Register a name for the handler's subscription in the event store
     name: "Posts.Projectors.Post",
-    application: Franklin.CommandedApplication,
-    # Ensure the database operation completes before allowing the command to be considered completed.
-    consistency: :strong
+    application: Franklin.CommandedApplication
```

Comment:
This change is unrelated to the ask of #1, but considering the wanted design goal, I made the change now.

If I am building out a CQRS app, with a large goal being education, I should avoid using the strong consistency option at all costs and embrace the async nature of projections.